test/types/helpers/helpers.types.ts

```diff
@@ -47,8 +47,6 @@ Handsontable.helper.isEdge();
 Handsontable.helper.isEmpty(1);
 Handsontable.helper.isFunction(1);
 Handsontable.helper.isIE();
-Handsontable.helper.isIE8();
-Handsontable.helper.isIE9();
```

Comment:
`isIE9` was reverted. Should we keep the test?

Reply:
It was reverted within https://github.com/handsontable/handsontable/pull/6625/commits/b620dae739c195a5d9d456fa964e4303507219ff#diff-6a04aedfd13e4e8a5cf26a05949f4eecR50